test/page/index.js

```diff
@@ -92,7 +92,7 @@ var params = {
   width: 600,
   heatMap: true,
   trackHeight: 20,
-  trackLegendLabel: '?',
+  trackLegendLabel: '<i>?</i>',
```

Comment:
`trackLegendLabel` is settings parameter. A user can change the value before initializing Oncogrid. The `<i>?</i>` just for testing the legend label.